.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
- [ ] **[OPTIONAL]** In your documentation, it would be nice if you referred the users with instructions on how to use the `.env.example` and `.env` files so they can understand how to set up the project easily.

Reply:
@TedLivist
Thank you for the suggestion. I've included instructions to create the `.env` file.